src/loaders/ImageLoader.js

```diff
@@ -73,7 +73,7 @@ class ImageLoader extends Loader {
 
 		if ( url.slice( 0, 5 ) !== 'data:' ) {
 
-			if ( this.crossOrigin !== undefined ) image.crossOrigin = this.crossOrigin;
+			if ( this.crossOrigin !== undefined && this.crossOrigin != null ) image.crossOrigin = this.crossOrigin;
```

Comment:
TBH, I don't understand the issue and thus not the purpose of the additional check. Can you please elaborate in more detail what you are doing?

It's probably best if you demonstrate the issue with a live example so it's clear what you want to fix. Use https://jsfiddle.net/g3atw6k5/ as a template.